src/modules/homepage/index.tsx

```diff
@@ -0,0 +1,430 @@
+import { Link } from 'react-router-dom';
+import DateTime from '../../shared-components/DateTime/index';
+import WifiIcon from '@mui/icons-material/Wifi';
+import SignalCellularAltIcon from '@mui/icons-material/SignalCellularAlt';
+import BatteryFullIcon from '@mui/icons-material/BatteryFull';
+import personImage from '../../assets/illustrations/05.png';
+import { ArrowDropDown } from '@mui/icons-material';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import CameraAltIcon from '@mui/icons-material/CameraAlt';
+import ArrowDownwardIcon from '@mui/icons-material/ArrowDownward';
+import ArrowUpwardIcon from '@mui/icons-material/ArrowUpward';
+import LocalMallIcon from '@mui/icons-material/LocalMall';
+import EventRepeatIcon from '@mui/icons-material/EventRepeat';
+import RestaurantIcon from '@mui/icons-material/Restaurant';
+import { ReactComponent as Graphic } from '../../assets/illustrations/graphic.svg';
+
+const StyledPageContentDiv = styled('div')({
+  overflow: 'auto',
+  maxHeight: 'calc(100vh - 70px)',
+});
+
+const StyledBottomNavigationContainer = styled('div')({
+  position: 'sticky',
+  bottom: 0,
+  left: 0,
+  right: 0,
+  width: '375px',
+  zIndex: '1000',
+  backgroundColor: '#fff',
+});
+
+const StyledRectangle = styled(Box)({
+  textAlign: 'center',
+  width: '375px',
+  height: '312px',
+  gap: '0px',
+  borderRadius: '0px 0px 32px 32px',
+  background:
+    'linear-gradient(183.33deg, #FFF6E6 9.56%, rgba(248, 237, 216, 0) 124.27%)',
+});
+
+export const Homepage = () => {
+  return (
+    <Stack sx={{ minHeight: '100vh', position: 'relative', width: '375px' }}>
+      <StyledRectangle>
+        <Box
+          display='flex'
+          justifyContent='space-between'
+          alignItems='center'
+          maxWidth='375px'
+          height='44px'
+          padding='0 16px'
+        >
+          <Typography variant='body2' fontWeight='700'>
+            <DateTime />
+          </Typography>
+          <Box display='flex' alignItems='center' gap='4px'>
+            <SignalCellularAltIcon fontSize='small' />
+            <WifiIcon fontSize='small' />
+            <BatteryFullIcon
+              fontSize='small'
+              sx={{ transform: 'rotate(90deg)' }}
+            />
+          </Box>
+        </Box>
+
+        <Box
+          display='flex'
+          justifyContent='space-between'
+          alignItems='center'
+          maxWidth='375px'
+          height='64px'
+          padding='0 16px'
+        >
+          <Box
+            sx={{
+              border: '2px solid #7F3DFF',
+              borderRadius: '50%',
+              padding: '2px',
+            }}
+          >
+            <Avatar alt='user image' src={personImage} />
+          </Box>
+
+          <Box display='flex' flexDirection='row' alignItems='center'>
+            <IconButton>
+              <ArrowDropDown />
+            </IconButton>
+            <Typography>October</Typography>
```

Comment:
This should be a Regular3 shared component. Since it's not available yet, you can create it in the `shared-components/Typography` folder. You'd need to allow the component to accept different color as input as it may use different colors.
#7F3DFF

Reply:
I added Regular2 shared component for typography. And it accept different styles as input.